code/modules/vehicles/cars/clowncar.dm

```diff
@@ -18,6 +18,14 @@
 	. = ..()
 	initialize_controller_action_type(/datum/action/vehicle/sealed/horn/clowncar, VEHICLE_CONTROL_DRIVE)
 
+/*
+/obj/vehicle/clowncar/vehicle_move(direction) //Prevent it from moving onto space
+	if(isspaceturf(get_step(src, direction)))
+		return FALSE
+	else
+		return ..()
+*/
+
```

Comment:
if this isn't relevant to the PR, do not add it

Reply:
F wrong button there.  I don't know how to remove these.

Reply:
Just edit it again.